mujoco_py/mjpid.pyx

```diff
@@ -80,21 +80,114 @@ cdef mjtNum c_pid_bias(const mjModel* m, const mjData* d, int id) with gil:
     d.userdata[id * NUM_USER_DATA_PER_ACT + IDX_LAST_ERROR] = error
     d.userdata[id * NUM_USER_DATA_PER_ACT + IDX_DERIVATIVE_ERROR_LAST] = derivative_error
     d.userdata[id * NUM_USER_DATA_PER_ACT + IDX_INTEGRAL_ERROR] = integral_error
-    
+
+    if effort_limit_low != 0.0 or effort_limit_high != 0.0:
+        f = fmax(effort_limit_low, fmin(effort_limit_high, f))
+    return f
+
+""" 
+   Inverse Dynamics (ID) Controller
```

Comment:
As per slack discussion, it might be more clear to change ID to something like InvDyn to avoid confusion between id for index, id for integral/derivative, and id for inverse dynamics.